libs/user-facing-errors/src/quaint.rs

```diff
@@ -36,168 +38,175 @@ pub fn invalid_connection_string_description(error_details: &str) -> String {
     details.replace('\n', " ")
 }
 
+// The following errors may arise from both native or external connections (TODO, check if this is true):
+// - `DatabaseDoesNotExist`
+// - `DatabaseAccessDenied`
+// - `DatabaseAlreadyExists`
+// - `TableDoesNotExist`
+// - `DatabaseUrlIsInvalid`
+//
+// TODO: try replacing `unreachable!` with `default_value`
+//
+// TODO: try handling `connection_info == ConnectionInfo::External(_)` right away
```

Comment:
These ones are all experiments to try to reduce the gzip'd bundle size.

Reply:
Note:
- "replacing `unreachable!` with `default_value`" results in 1 additional gzip'd byte

Reply:
The following errors can indeed happen when using the `pg` driver adapter:
- `DatabaseDoesNotExist`
- `DatabaseAccessDenied`
- `DatabaseAlreadyExists `, but one would need to run e.g. "await prisma.$executeRaw`CREATE DATABASE tests"
- `TableDoesNotExist`, which is only possible when using `prisma.$executeRaw`